processing/data_collection/gazette/spiders/ma_sao_luis.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+import dateparser
+from datetime import datetime
+from scrapy_splash import SplashRequest
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+from scrapy.shell import inspect_response
+
+
+class MaSaoLuisSpider(BaseGazetteSpider):
+    MUNICIPALITY_ID = '2111300'
```

Comment:
Change it to TERRITORY_ID
See: https://github.com/okfn-brasil/diario-oficial/commit/89452457665e0b4c76e94ab0aa63be21f8be39df